packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -28,7 +28,8 @@ class AndroidWorkflow extends Workflow {
     androidValidator.addValidator(new Validator(
       'Android SDK',
```

Comment:
I updated it to read `Android Studio / Android SDK`. That way we're not checking for the sdk, but telling people that studio is installed.